api/kessel/inventory/v1beta2/resource.proto

```diff
@@ -12,7 +11,9 @@ option java_package = "org.project_kessel.api.inventory.v1beta2";
 message Resource {
  string inventory_id = 1;
  string resource_type = 2 [(buf.validate.field).string = {min_len: 1}];
- ReporterData reporter_data = 3 [(buf.validate.field).required = true];
- google.protobuf.Struct common_resource_data = 4 [(buf.validate.field).required = true, json_name = "commonResourceData"];
+ string reporter_type = 3 [(buf.validate.field).string = {min_len: 1}];
+ string reporter_instance_id = 4 [(buf.validate.field).string = {min_len: 1}];
+ ResourceRepresentations resource_representation = 7 [(buf.validate.field).required = false, json_name = "resourceRepresentation"];
```

Comment:
Could we get by with just `representations` instead of `resource_representations`/`"resourceRepresentations" (which is a bit verbose)?

Also, I think it should be plural.